patches/tModLoader/Terraria/ModLoader/ModContent.cs

```diff
@@ -103,6 +103,8 @@ public static bool FileExists(string name)
 	/// <summary>
 	/// Gets the asset with the specified name. Throws an Exception if the asset does not exist.
 	/// <para/>
+	/// Can be safely called on the server despite the server not loading assets, so there is no need to add checks for <see cref="Main.dedServ"/>.
+	/// <para/>
```

Comment:
Belongs in the inherited `IAssetRepository.Request` docs. Perhaps with a note that on the server assets will always be returned as if loaded via `DoNotLoad`.